SubZero/src/model/filter/RGBFilter.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * RGBFilter.cpp
+ *
+ *  Created on: Feb 24, 2015
+ *      Author: ahsueh1996
+ */
+
+#include "RGBFilter.h"
+
+RGBFilter::RGBFilter(int r, int g, int b) {
+	setValues(r,g,b);
+}
+
+RGBFilter::~RGBFilter() {
+	// TODO Auto-generated destructor stub
+}
+
+int RGBFilter::filter (CamData * inputImg) {
+	return 0;
+}
+
+
+void RGBFilter::setValues(int r, int g, int b){
+	this -> r = r;
+	this -> g = g;
+	this -> b = b;
+}
```

Comment:
Awesome! We talked about this during Model's meeting on Tuesday. I will fix my code for the next pull request.